src/www/views/servers_view/server_list_item/server_card/index.ts

```diff
@@ -35,6 +35,15 @@ const sharedCSS = css`
     all: initial;
   }
 
+  * {
+    -webkit-user-select: none;
+    -moz-user-select: none;
```

Comment:
We only need to worry about Android and iOS webviews and chrome. I guess that's all Webkit?